Tests/test_pile_ui.gd

```diff
@@ -0,0 +1,44 @@
+extends GutTest
+#test for ui pop up
+
+var _card_container_scene: PackedScene = load("res://Cards/CardContainer.tscn")
+var _card_container_scroll_scene: PackedScene  = load("res://#Scenes/CardScrollUI.tscn")
+var _card_container: CardContainer = null
+var _card_scroll: Control = null
+
+const SCROLLCONTAINER = 2
+const GRIDCONTAINER = 0
```

Comment:
Maybe a nitpick here, but I do like to separate words with `_` so in this case `SCROLL_CONTAINER` and `GRID_CONTAINER`

Reply:
yes indeed ! I didn't explicitly add it in the `CONTRIBUTING.md` in the formatting part, but this is the format of the [GDScript style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#trailing-comma)